This version of Mako uses old setup.cfg file, which is no longer valid and thus
prevents tests from running. Newer versions have this already fixed/updated.

--- Mako-1.0.4/setup.cfg
+++ Mako-1.0.4/setup.cfg
@@ -3,7 +3,7 @@ tag_build =
 tag_date = 0
 tag_svn_revision = 0
 
-[pytest]
+[tool:pytest]
 addopts = --tb native -v -r fxX
 python_files = test/*test_*.py
 
